toml.md

```diff
@@ -103,10 +103,9 @@ Punctuation, spaces, arrows, box drawing and private use characters are not
 allowed. Note that bare keys are allowed to be composed of only ASCII digits,
 e.g. 1234, but are always interpreted as strings.
 
-ℹ️ The exact ranges of allowed code points can be found in the 
+ℹ️ The exact ranges of allowed code points can be found in the
```

Comment:
@abelbraaksma FYI if you want the 'info' emoji to show up consistently as the actual emoji and not a crap outline version
you need to follow it with `&#xfe0f;` version

Ref: https://stackoverflow.com/questions/72048059/how-to-specify-emoji-version-of-a-unicode-character-in-html

Reply:
Ah, yes, that makes sense. Fixed.